src/infrastructure/content/ContentCommunicator.java

```diff
@@ -1,24 +1,145 @@
 package infrastructure.content;
 
+import org.json.JSONObject;
+import networking.CommunicatorFactory;
+import networking.ICommunicator;
+import java.util.HashMap;

```

Comment:
You will need to import LanCommunicator and utility.ClientInfo from networking as well. They are needed for the networking factory to work.

Reply:
LanCommunicator implements ICommunicator which is all that we need. When we call any method, the handle goes to the class in which that method is declared. We don't need to import anything extra. That class will take care of those extra imports.

Reply:
Alright